css/style.css

```diff
@@ -470,3 +470,672 @@
+  background: #f00;
+  background-size: 100%;
+   
+}
+
+.frame__title {
+  display: flex;
+  width: 316px;
+  height: 74px;
+  margin: 51px 0 0 56px;
+}
+
+.frame__item {
+  display: flex;
+  width: 316px;
+  height: 134px;
+  margin: 12px 0 49px 56px;
+}
+
+/* Оценка защиты */
+
+.frame__title-rating {
+  display: flex;
+  width: 263px;
+  height: 37px;
+  margin: 0px 0 29px 56px;
+}
+
+.frame__result {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+  width: 295px;
+  height: 173px;
+  font-size: 17px;
+  font-weight:bold;
+  line-height: 21px;
+  list-style:none;
+  margin: 0px 0 101px 56px;
+}
+
+.result-left {
+  display: flex;
+  width: 151px;
+  height: 173px;
+  padding:0;
+  margin:0;
+}
+
+li { list-style-type: none; }
+
+.result-left__list {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding:0;
+  margin:0;
+}
+
+.list__item {
+  display: flex;
+  justify-content:space-between;
+  width: 119px;
+  height: 21px;
+  text-transform: uppercase;
+}
+
+.list__item-rack {
+  width: 53px;
+  height: 13px;
+  font-size: 10px;
+  font-weight:bold;
+  line-height: 12px;
+  color: #fcc;
+  margin: 6px 0 0 0;
+  text-transform: uppercase;
+}
+
+.result-right {
+  display: flex;
+  width: 140px;
+  height: 173px;
+  padding:0;
+  margin:0;
+}
+
+.result-right__list {
+  display: flex;
+  flex-flow: column wrap;
+  justify-content:space-between;
+  padding:0;
+  margin:0;
+}
+
+.result-right__item {
+  display: flex;
+  flex-flow: row wrap;
+  align-content: center;
+  justify-content:space-between;
+  width: 138px;
+  height: 25px;
+  padding:0;
+  margin:0;
+}
+
+.item__star {
+  display: flex;
+  width: 25px;
+  height: 25px;
+  background: url(https://i.ibb.co/BfqgXGC/Star-1.png) no-repeat;
+  margin: 0 2px 0 0;
+}
+
+/* Результат оценки защиты */
+
+.frame__five-star {
+  display: flex;
+  flex-flow: row wrap;
+  width: 290px;
+  height: 92px;
+  margin: 0 0 50px 66px;
+}
+
+.frame__image-shield {
+  width: 62px;
+  height: 74px;
+  background: url(https://i.ibb.co/jy99Qdg/Group-5star.png) no-repeat;
+  margin: 9px 0 0 0;
+}
+
+.frame__five-text {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: flex-start;
+  align-content: center;
+  width: 200px;
+  height: 92px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 50px;
+  line-height: 43px;
+  margin:0 0 0 23px;
+}
+
+.frame__five-text_safety {
+  width: 187px;
+  height: 31px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 25px;
+  line-height: 30px;
+  margin:0;
+}
+
+/* ссылка тест и кнопка заказать */
+
+.information-or-order {
+  width: 347px;
+  height: 40px;
+}
+
+.frame__info {
+  display:flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  width: 347px;
+  height: 40px;
+  margin-left: 38px;
+}
+
+.info__image {
+  display: flex;
+  width: 22px;
+  height: 22px;
+  background: url(https://i.ibb.co/zFFHDgv/Group-15.png) no-repeat;
+  background-size: 100%;
+}
+
+.info__link {
```

Comment:
Codacy found an issue: [Expected selector ".info__link" to come before selector ".info__link:hover" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/Hi-Tesla/pullRequest?prid=5726736)